cmd/storagenode/entrypoint

```diff
@@ -1,6 +1,14 @@
 #!/bin/sh
 set -euo pipefail
 
+if [[ ! -f "${IDENT_PATH}/identity.key" ]]; then
+  cd /tmp
+  wget https://storj-v3-alpha-builds.storage.googleapis.com/e82d336-go1.11/identity_linux_amd64.zip
+  unzip identity_linux_amd64.zip
+  mv identity_linux_amd64 identity
+  /tmp/identity create --difficulty 5 storagenode
```

Comment:
this concerns me for three reasons -

first, didn't we just copy the identity binary we compiled in?
secondly, this is way too low of a difficulty and identities shouldn't get generated automatically (ssh keys aren't).
third, if this works, this implies that we're not requiring signed certificates

Reply:
I think Bender is on it to do exactly that.

Reply:
Updated default difficulty to 30 with a way to configure it. This enables testing in dev environment until we take the time to add high difficulty certs and enforce signing.